src/Microsoft.AspNet.Owin/OwinFeatureCollection.cs

```diff
@@ -155,6 +155,16 @@ void IHttpResponseFeature.OnSendingHeaders(Action<object> callback, object state
             register(callback, state);
         }
 
+        void IHttpResponseFeature.OnResponseCompleted(Action<object> callback, object state)
```

Comment:
You may also want to expose it in the OWIN environment like `OnSendingHeaders`: https://github.com/aspnet/HttpAbstractions/blob/response-feature/src/Microsoft.AspNet.Owin/OwinEnvironment.cs#L64